Makefile

```diff
@@ -121,6 +125,14 @@ dist-offline: clean-build clean-pyc
 	python setup.py bdist_prestoadmin
 	ls -l dist
 
+docker-dist: docker-dist-online
+
+docker-dist-online: clean-build clean-pyc
+	docker run --rm -v `pwd`:/workdir -w /workdir -u `id -u`:`id -g` teradatalabs/presto-admin-devenv:1 make dist-online
```

Comment:
Can you please update the readme to cover the following:

1. When you'd want to run dist-<online|offline> vs docker-dist<online|offline>
2. What python gets used to build the installers when you make any of the above targets
3. Where the installer ends up.

Thanks!